### PR TITLE
Structure branch

### PR DESCRIPTION
**Pull request by Chike1990**
Awesome Books is a basic website that allows users to add/remove books from a list. This is achieved using plain HTML, JavaScript objects, arrays, DOM, and basic events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
-# Awesome_books
\ No newline at end of file
+# Awesome Books App
+
+Awesome Books is a basic website that allows users to add/remove books from a list. This is achieved using plain HTML, JavaScript objects, arrays, DOM, and basic events.
+
+
+![Screenshot](img\addBookUI.png)
+![Screenshot](img\appPage.png)
```
Comment (abruzy) [inline README.md]: Good job 👍🏽 

- [ ] fix the image to show in the preview

File: script.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  removeBtn.addEventListener('click', () => {
+  // removes from array
+    const bookIndex = myBook
+      .map((book) => book.title)
+      .indexOf(newBook.title);
+    myBook.splice(bookIndex, 1);
+    // remove book from local storage
+    localStorage.removeItem(bookIndex);
+  });
+}
```
Comment (abruzy) [inline script.js]: Good job here 👍🏽 

- [x] create a function ```removeBook(id)``` and pass it down to the event listener like this ```document.addEventListener('click', removeBook)```

File: script.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+function displayBook(newBook) {
+  const list = document.querySelector('#book-list');
+  const row = document.createElement('div');
+  row.innerHTML = `
+    <h4>${newBook.title}</h4>
+    <h4>${newBook.author}</h4>
+    <button><a href="#" class="btn btn-danger btn-sm remove">Remove</a></button><hr>`;
+
+  list.appendChild(row);
```
Comment (abruzy) [inline script.js:22]: Good job 👍🏽 

- [OPTIONAL] 
the use of `DOMContentLoaded` over `load` "A different event, load, should be used only to detect a fully-loaded page. It is a common mistake to use load where DOMContentLoaded would be more appropriate.", [see](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event )

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @Chike1990  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 
_1. Kindly check the comments under the review._
_2. Ensure to also add link to the live demo of the application._

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Chike1990 @ccobasi  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**